numpy/core/tests/test_numeric.py

```diff
@@ -11,7 +11,7 @@
 from numpy.testing import (
     assert_, assert_equal, assert_raises, assert_raises_regex,
     assert_array_equal, assert_almost_equal, assert_array_almost_equal,
-    assert_warns, HAS_REFCOUNT
+    assert_warns, assert_allclose, HAS_REFCOUNT
```

Comment:
Do you still need the `assert_allclose` now that you are not returning python floating point numbers? Rounding should be defined very closely, but I guess it can have minor errors. How about using `assert_array_max_ulp`?

Reply:
> `assert_array_max_ulp`

This works! Did not know about this, thanks.